src/webapp/components/table/performance-overview-table/PerformanceOverviewTable.tsx

```diff
@@ -0,0 +1,277 @@
+import React, { useEffect, useMemo, useState } from "react";
+import _ from "lodash";
```

Comment:
can we try using generic\Collection instead of lodash? Its ESTs version of lodash

Reply:
I use `Collection` now, and use `import _ from "./Collection";` 
How can I both Collection and HashMap? Should I them under they name?
```ts
import collection from "./Collection";
import hashmap from "./HashMap";
```